README.md

```diff
@@ -535,26 +535,6 @@ PaperTrail.request.disable_model(Widget)
 This setting, as with all `PaperTrail.request` settings, affects only the
 current request, not all threads.
 
-#### Per Method
-
-You can call a method without creating a new version using `without_versioning`.
- It takes either a method name as a symbol:
-
-```ruby
-@widget.paper_trail.without_versioning :destroy
-```
-
-Or a block:
-
-```ruby
-@widget.paper_trail.without_versioning do
-  @widget.update_attributes name: 'Ford'
-end
-```
-
-During `without_versioning`, PaperTrail is disabled for the whole model
-(e.g. `Widget`), not just for the instance (e.g. `@widget`).
-
```

Comment:
Removing this is good, thanks. But, do you feel like writing up something new to replace it? Here's the old deprecation warning for reference:

```
    DPR_WITHOUT_VERSIONING = <<-STR
      my_model_instance.paper_trail.without_versioning is deprecated, without
      an exact replacement. To disable versioning for a particular model,
      ```
      PaperTrail.request.disable_model(Banana)
      # changes to Banana model do not create versions,
      # but eg. changes to Kiwi model do.
      PaperTrail.request.enable_model(Banana)
      ```
      Or, you may want to disable all models,
      ```
      PaperTrail.request.enabled = false
      # no versions created
      PaperTrail.request.enabled = true
      # or, with a block,
      PaperTrail.request(enabled: false) do
        # no versions created
      end
      ```
    STR
```

Reply:
Not a problem 👍done. Any thoughts RE my comment on `PaperTrail.controller_info` ?